### ISSUE TITLE
Label Paper S09

### ISSUE THREAD

**Issue by kerautret** [RLPR/LabelReviews#7]
# General info
- [x] Conference: ICPR 2018
- [x] Papers title: Deep Difference Analysis in Similar-looking Face recognition
- [x] GitHub repo:  https://github.com/zhongyy/Deep-Difference-Analysis-in-Similar-looking-face-recognition
- [x] Submission date: 30/07/2018
- [ ] Platform: 
- [ ] Language:
- [ ] Results to be reproduced: (indicates reference to figure/tab) 
- [ ] Reproduction main step instructions:
       `Give the main steps to reproduce the figure/tab` 

# Reviewer feedback
 - [ ] Tested platform: 
 - [ ]  Result reproduced: 
 - [ ]  Reviewer comment:

## Details Results
- Execution without errors:
    - [ ] without changes
    - [ ] with small changes
    - [ ] with a lot of changes
- Results are obtained:
    - [ ] always
    - [ ] sometimes
    - [ ] never
- Results correspond to the paper's figures/tables:
    - [ ] exactly
    - [ ] with negligible differences
    - [ ] with differences
    - [ ] with a lot of differences

**Comment by kerautret** [comment]
@shidahe Could you please add the command description to reproduce the results ? (to complete the review form and simplifiante the reviewing work)

**Comment by kerautret** [comment]
@zhongyy Could you please complete the form ?

- [ ] Platform: 
- [ ] Language:
- [ ] Results to be reproduced: (indicates reference to figure/tab) 
- [ ] Reproduction main step instructions:
       `Give the main steps to reproduce the figure/tab` 
and send us in private the pdf file of ICPR  ? (rrpr.chairs@inria.fr)
Thanks

**Comment by zhongyy** [comment]
- [  ] Platform: Ubuntu 14.04.5 LTS
 - [  ]  Language: MATLAB R2014b
 - [  ] Results to be reproduced: Figure 2, Figure 4
 - [  ] Reproduction main step instructions:
### Install caffe
1. Install [caffe](https://github.com/BVLC/caffe).
2. compile caffe and matcaffe (matlab wrapper for caffe)
```
make all -j4
make matcaffe
```
### Download the code and model
1. Download the code.
```
git clone https://github.com/zhongyy/Deep-Difference-Analysis-in-Similar-looking-face-recognition.git
```
2. Download the pretrained model and put it in the root fold.

The model could be download from [Baiduyun](https://pan.baidu.com/s/19FHYCmDCAKjgNmJVMOvJgA). password: dmo1

### run the demo

#### We provide an example of Chow Yun Fat and a stand-in for him. Using the code, you get the result shown in paper Figure 2.

1. Extract the features of Face images.

As we provide the related images and in-process data, you could run extract_zhou.m and then run generate_zhou.m to get the result from scratch.

Note that some code related to your own caffe path should be changed.
```
matlab extract_zhou.m
matlab generate_zhou.m
```
2. If it is hard for you to run feature extraction code related to Matcaffe, you can run generate_zhou.m using some in-process data.
```
matlab generate_zhou.m
```

### To reproduce Figure 4 in the paper

#### We provide the aligned images. This result is shown in paper Figure 4.

1. Extract the features of Face images.

As we provide the related images and in-process data, you could run extract.m and then run generate.m to get the result from scratch.

Note that some code related to your own caffe path should be changed.
```
matlab extract.m
matlab generate.m
```
2. If it is hard for you to run feature extraction code related to Matcaffe, you can run generate.m using some in-process data.
```
matlab generate.m
```

**Comment by zhongyy** [comment]
I have email the ICPR 2018 paper 《Deep Difference Analysis in Similar-looking Face recognition》. Please let me know if there is anything wrong. Thank you a lot!

**Comment by kerautret** [comment]
@zhongyy thanks a lot for the complement 👌I move it in the first form.

**Comment by zhongyy** [comment]
Thank you！

**Comment by akrah** [comment]
Dear @zhongyy,
I started the review but I can't download the pre-trained model from Baiduyun for two reasons:
    - the site is in Chinese
    - i don't know (or it is impossible) to select the folder `ICPR_model`

Concerning the Chinese, the problem is that it cannot be easily translated in other languages due to the password: by using google translate, i'm able to translate the first page asking the password but a click to access to the model has no effect.

So, is it possible to store your model on a english website or with a direct link?

**Comment by akrah** [comment]
After some other tests, new conclusion: I'm able to select the file with Chrome (not with Firefox).
But the site ask me to download a software to be able to download the files, software only available on Windows.So, please, could you upload your model on a website less restrictive? Thanks!

**Comment by zhongyy** [comment]
I am not familiar with the process of sharing big files, sorry for making something wrong. I have put the pre-trained model on  [Google drive](https://drive.google.com/drive/folders/1nGv8---4SqeoRNYIIvun0yPsqYXPYHEI?usp=sharing).  Would you check it once again? Please let me known if there is still something wrong. Thank you!

**Comment by akrah** [comment]
The new link works fine, thanks!
